provisioning_templates/snippet/kickstart_ifcfg_generic_interface.erb

```diff
@@ -41,6 +43,10 @@ DEFROUTE=<%= primary %>
 <%-     @subnet.dns_servers.each_index do |index| -%>
 <%=       "DNS#{index +1}=\"#{@subnet.dns_servers[index]}\"" %>
 <%-     end -%>
+<%-   elsif !@dhcp && @subnet6 -%>
+<%-     @subnet6.dns_servers.each_index do |index| -%>
+<%=       "DNS#{index +1}=\"#{@subnet6.dns_servers[index]}\"" %>
```

Comment:
This inded starts at 0 again so it should be:
```suggestion
<%=       "DNS#{index + @subnet.dns_servers.length + 1}=\"#{@subnet6.dns_servers[index]}\"" %>
```

Of course this also needs some check in case there is no IPv4 subnet.

Reply:
I am not sure what you mean, DNS entries do start with 1 according to the man page: https://www.systutorials.com/docs/linux/man/5-nm-settings-ifcfg-rh/

I'd rather not touch this, I've only copied this from IPv4 section and I would like to avoid re-testing this again if possible. It works just like this.

Reply:
My bad, I missed the `elsif` and thought you could have both IPv4 and IPv6 DNS servers. Airport reviews are not the best reviews.